src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs

```diff
@@ -143,6 +154,20 @@ private void StopServiceCallback(object sender, EventArgs e)
             }
         }
 
+        private void DisableAsyncCompletionCallback(object sender, EventArgs e)
+        {
+            if (!_disableAsyncCompletionMenuCmd.Enabled)
+            {
+                return;
+            }
+
+            var componentModel = ServiceProvider.GetService<SComponentModel, IComponentModel>();
+            var featureServiceFactory = componentModel.GetService<IFeatureServiceFactory>();
+            featureServiceFactory.GlobalFeatureService.Disable(PredefinedEditorFeatureNames.AsyncCompletion, controller: null);
```

Comment:
passing in null will throw. You need to pass an object which implements IFeatureController (empty interface, so you can slap it on anything). 
This is to track who disabled a feature so that it's not accidentally re-enabled if multiple parties are present.

Reply:
Fixed